dialect.go

```diff
@@ -50,6 +50,8 @@ func SetDialect(s string) error {
 		d = dialect.Vertica
 	case "ydb":
 		d = dialect.Ydb
+	case "turso":
```

Comment:
```
case "sqlite3", "sqlite", "turso":
```

This would likely suffice since the dialect is the same, makes it a lot easier to maintain  🙂

Reply:
Addressed here: https://github.com/pressly/goose/pull/658#discussion_r1428738176